api-module-library/slack/models/integrationMapping.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 const mongoose = require('mongoose');
-const { IntegrationMapping: Parent } = require('@friggframework/module-plugin');
+const { IntegrationMapping: Parent } = require('@friggframework/integrations');
 
 const schema = new mongoose.Schema({});
 
-const name = 'SlackMessage';
+const name = 'WorkflowConfig';
```

Comment:
Ah, I had _totally_ spaced on the fact that this was in the **Slack** API Module. My apologies, @debbie-yu 

This makes a ton of sense to include here for folks to use as reference. Message and Channels seem like things devs will need to track.

Relatedly, the Ironclad API Module should get Workflow or probably properly `IroncladWorkflow` (vs `WorkflowConfig`, to match the pattern of "api/type plus the object being mapped/tracked)